CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ All notes should be prepended with the location of the change, e.g. `(proto)` or
 ### Fixed
 
 - (resolver) Malformed label: -- when parsing resolv.conf #1985 by Jake-Shadle
-- (proto) Fix truncation for UDP #1975 by nmittler 
+- (proto) Fix truncation for UDP #1975 by nmittler
```

Comment:
are all of these CHANGELOG updates intentional?

Reply:
Nope, reverted these changes.